node/src/chain_spec.rs

```diff
@@ -116,32 +148,34 @@ pub fn local_testnet_config(id: ParaId) -> ChainSpec {
 		"canvas_local",
 		ChainType::Local,
 		move || {
-			// TODO [#76]: Right now these are being left as default values, but in the old `canvas-node`
-			// we had some non-default authorities, as well as accounts for the faucent and
-			// `ink-waterfall`. We'll need to add those back in
 			testnet_genesis(
-				get_account_id_from_seed::<sr25519::Public>("Alice"),
-				vec![get_from_seed::<AuraId>("Alice"), get_from_seed::<AuraId>("Bob")],
+				local_testnet_root(),
+				local_testnet_authorities(),
 				vec![
-					get_account_id_from_seed::<sr25519::Public>("Alice"),
-					get_account_id_from_seed::<sr25519::Public>("Bob"),
-					get_account_id_from_seed::<sr25519::Public>("Charlie"),
-					get_account_id_from_seed::<sr25519::Public>("Dave"),
-					get_account_id_from_seed::<sr25519::Public>("Eve"),
-					get_account_id_from_seed::<sr25519::Public>("Ferdie"),
-					get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+					local_testnet_root(),
+					// AccountId of the Canvas Testnet faucet
+					hex!("18c64aa111a8a0e6e4eed41d6d906c7614d745e48be3cfc13b6128e1d51f4405").into(),
+					// AccountId of an account which `ink-waterfall` uses for automated testing
+					hex!("0e47e2344d523c3cc5c34394b0d58b9a4200e813a038e6c5a6163cc07d70b069").into(),
```

Comment:
@riusricardo Do you know if it's even possible to have endowed accounts if we use the Relaychain currency?

Reply:
We can but we don't do it now (Statemint isn't connected to Rococo). The consequence of doing it is that we'll be minting extra native tokens from the relay chain on the parachain. However we can do it for Rococo and put a big warning/TODO so we don't forget changing it when we create the other runtimes.

Reply:
We can but we don't do it now (Statemint isn't connected to Rococo). The consequence of doing it is that we'll be minting extra native tokens from the relay chain on the parachain. However we can do it for Rococo and put a big warning/TODO so we don't forget changing it when we create the other runtimes.